code/modules/clothing/head/misc.dm

```diff
@@ -21,13 +21,48 @@
 	icon_state = "pwig"
 	inhand_icon_state = "pwig"
 
+#define RABBIT_CD_TIME 30 SECONDS
+
 /obj/item/clothing/head/that
 	name = "top-hat"
 	desc = "It's an amish looking hat."
 	icon_state = "tophat"
 	inhand_icon_state = "that"
 	dog_fashion = /datum/dog_fashion/head
 	throwforce = 1
+	/// Cooldown for how often we can pull rabbits out of here
+	COOLDOWN_DECLARE(rabbit_cooldown)
+
+/obj/item/clothing/head/that/attackby(obj/item/W, mob/user, params)
+	. = ..()
+	if(istype(W, /obj/item/gun/magic/wand))
```

Comment:
The mime has access to a wand of nothing, do they not? This doesn't require wizard interaction. (Should doing this use a charge on the wand?)

Reply:
It wasn't intended to require wizard interaction, and I also think it would be exceedingly lame to sacrifice wand charges for a bunny